prody/dynamics/plotting.py

```diff
@@ -211,7 +211,7 @@ def showProjection(ensemble, modes, *args, **kwargs):
       * 3 modes: :meth:`~mpl_toolkits.mplot3d.Axes3D.plot`"""
 
     import matplotlib.pyplot as plt
-
+    plt.figure()
```

Comment:
I think it depends also which plotting tools we use as to whether there should be an option on this. For example, pcolor makes a mess rather than nicely plotting on top. This is particularly horrible when you include color bars as these then get added next to each other.